modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/Tabs/BasicInfo/SearchableContainer.tsx

```diff
@@ -24,6 +24,7 @@ const languages = Liferay.Language.available;
 const languageLabels = Object.values(languages).map((language) => {
 	return {label: language};
 });
+const defaultLanguage = Object.values(languageLabels[0]).toString();
```

Comment:
Hi @vngoliveira, I think you can achieve the same result with  `const defaultLanguage = languageLabels[0].label;` and it's simpler.